coverage/artifacts/artifacts_test.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package artifacts
```

Comment:
I usually found that empty test packages are misleading: the don't test anything, but lead people to think they do. I would suggest an all-or-nothing approach here (and everywhere): either write tests or don't add an empty package. If acceptable, at least a TODO with an issue number should be added to the empty package.

Reply:
removed